api/src/main/resources/context.json

```diff
@@ -0,0 +1,8 @@
+{
+  "@context": {
+    "@base": "http://id.wellcomecollection.org/",
+    "@vocab": "http://wellcomecollection.org/ontologies/works/",
```

Comment:
By default any new stuff will be HTTPS – is it a hard requirement that these are HTTP?